paddle/fluid/framework/data_type_transform_test.cc

```diff
@@ -189,4 +194,120 @@ TEST(DataTypeTransform, CPUTransform) {
                 static_cast<paddle::platform::float16>(in_data_bool[i]).x);
     }
   }
+
+  // data type transform from/to bfloat16
+  {
```

Comment:
The previous block could be parameterized as a function and reused here, to avoid duplication of code.
This can be done later as a refactoring.

Reply:
You are right. I will refactor it later.